.github/workflows/ci.yaml

```diff
@@ -46,29 +50,33 @@ jobs:
           cache-downloads-key: downloads-${{ steps.date.outputs.date }}
           create-args: >-
             python=${{ matrix.python-version }}
-
       - name: "Install"
         run: pip install --no-deps -e .
 
       - name: "Test imports"
         run: |
           # if we add more to this, consider changing to for + env vars
           python -Ic "import kartograf"
+
       - name: "Environment Information"
         run: |
           micromamba info 
           micromamba list
+
       - name: "Run tests"
         run: |
           pytest -n auto -v --cov=kartograf --cov-report=xml
+
       - name: "Test Notebooks"
+        if: ${{ matrix.os != 'macos' }}
```

Comment:
macos runners are super slow for nbval, there's probably no need to run notebook tests for them